notebooks/train_notebook.ipynb

```diff
@@ -0,0 +1,1144 @@
+      "execution_count": 2,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# # connect to our wandb project\n",
+        "# !pip install wandb\n",
+        "# !wandb login \"API_KEY\""
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "**Imports**"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 3,
+      "metadata": {
+        "id": "Ow3lduSvGwrU"
+      },
+      "outputs": [
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "/home/alodie/miniconda3/envs/avalanche-dev-env/lib/python3.8/site-packages/tqdm/auto.py:22: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
+            "  from .autonotebook import tqdm as notebook_tqdm\n"
+          ]
+        }
+      ],
+      "source": [
+        "import torch\n",
+        "import torch.nn as nn\n",
+        "import random\n",
+        "import sys \n",
+        "sys.path.append(\"..\")\n",
+        "import time\n",
+        "import gc"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 4,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "NJaOta3cmQwI",
+        "outputId": "81ddc041-d1ee-410d-ab80-e9a75890b761"
+      },
+      "outputs": [],
+      "source": [
+        "# from google.colab import drive\n",
+        "# drive.mount('/content/drive')"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "**Setup**"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 5,
+      "metadata": {
+        "id": "c2CF9rXbGwrU"
+      },
+      "outputs": [],
+      "source": [
+        "SEED = 1234\n",
+        "\n",
+        "random.seed(SEED)\n",
+        "torch.manual_seed(SEED)\n",
+        "torch.cuda.manual_seed(SEED)\n",
+        "torch.backends.cudnn.deterministic = True\n",
+        "\n",
+        "device = torch.device(\"cuda:1\" if torch.cuda.is_available() else \"cpu\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "**Data**"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 6,
+      "metadata": {
+        "id": "JxwCl4lcGwrW"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "Files already downloaded and verified\n",
+            "Files already downloaded and verified\n",
+            "Files already downloaded and verified\n"
+          ]
+        }
+      ],
+      "source": [
+        "from src.data import get_transformed_data, make_data_loaders\n",
+        "from src.transforms import make_transforms\n",
+        "\n",
+        "BATCH_SIZE = 256\n",
+        "EPOCHS  = 100\n",
```

Comment:
The notebook currently has the following 3 lines:
1. EPOCHS  = 100
2. wandb.init(    config={        "epochs": EPOCHS,
3. EPOCHS  = 200

Option 1): delete 1., and move 2. to after 3., and re-run the 200 epochs
Option 2): merge 3. into 1., and do not re-run the 200 epochs (I'd be fine with this option)